pyproject.toml

```diff
@@ -143,18 +140,21 @@ ignore = [
     "C901",
 ]
 select = [
-    "C",
-    "E",
-    "F",
-    "I",
-    "PIE",
-    "Q",
-    "RET",
-    "RSE",
-    "SIM",
-    "TCH",
-    "TID",
-    "W",
+    # "B",  # flake-8-bugbear
+    "C",  # pylint-convention
+    "E",  # pycodestyle-error
+    "F",  # pyflakes
+    "I",  # isort
+    # "NPY",  # numpy
```

Comment:
A few extra linting jobs that would be nice to add, with labels of what these are